plugins/modules/deb_repository.py

```diff
@@ -77,7 +91,8 @@
 
 try:
     from pulp_glue.deb import __version__ as pulp_glue_deb_version
-    from pulp_glue.deb.context import PulpAptRepositoryContext
+    from pulp_glue.deb.context import PulpAptRepositoryContext, PulpAptRemoteContext
+    PulpAptRepositoryContext.NULLABLES.add("remote")
```

Comment:
One very last request:
Since we are monkeypatching glue here, let us add a bit of context for the future:


```suggestion

    # In pulp-glue <0.35 this is needed for idempotent removal of the remote.
    PulpAptRepositoryContext.NULLABLES.add("remote")
```

It should tell us when it's ok to remove this bandaid.

Reply:
Done